test/unit/server/rpc/procedures/google-maps.spec.js

```diff
@@ -58,13 +58,43 @@ describe('staticmap', function() {
         };
 
         it('should handle wraparound at map boundaries', function(){
-            let coords = staticmap._rpc._coordsAt(-170, 90, map);
+            let coords = googlemaps._rpc._coordsAt(-170, 90, map);
             assert(coords.lon > -180 && coords.lon < 180);
             map.center.lon = +150;
-            coords = staticmap._rpc._coordsAt(170, 90, map);
+            coords = googlemaps._rpc._coordsAt(170, 90, map);
             assert(coords.lon > -180 && coords.lon < 180);
         });
 
     });
 
+    describe('getGoogleParams', function() {
+
+        const opts = {
+            center: {
+                lat: 36.2645738345627,
+                lon: -82.54322767345267,
+            },
+            width: (640 / 1),
```

Comment:
Why is this divided by 1?

Reply:
it is showing division by scale which is one (copied from the service)